tests/api_tests/getUser.spec.ts

```diff
@@ -5,3 +5,10 @@ test.skip('should check response status', { tag: ['@api'] }, async ({ request })
     console.log(JSON.stringify(response));
     expect(response['_initializer'].status).toBe(200);
 });
+test('Assert 401 response status', async ({request}) =>{
```

Comment:
I would change the name of the test. You write the negative test for getting of list of users?

Reply:
done,
please review

Reply:
Please improve test name (make it focusing on what is being tested)